source/Shader.cpp

```diff
@@ -43,8 +43,16 @@ Shader::Shader(const char *vertex, const char *fragment)
 	
 	GLint status;
 	glGetProgramiv(program, GL_LINK_STATUS, &status);
-	if(status == GL_FALSE)
+	if(status == GL_FALSE) {
```

Comment:
Please match the style of the existing code

(braces on newline when following parentheses, no `std::` prefix when not required)

Reply:
What tool do I use to apply your formatting style? I haven't seen any clang-format rules and the reply hardly looks like the default one.

Reply:
There are no automated tool rules for the conventions used by this project. We do have editorconfig support for the whitespace requirements.

Other than that, there is the style guide:  https://endless-sky.github.io/styleguide/styleguide.xml

Reply:
Well, that's quite surprising. How about clang-format then?

Reply:
```suggestion
	if(status == GL_FALSE)
	{
```